### ISSUE TITLE
Swagger site not reachable after update to version 0.9 of the extension

### ISSUE THREAD

**Issue by philipp985** [sjkp/letsencrypt-siteextension#287]
After Updating Azure Let's Encrypt to Version 0.9.0 the swagger site is not available anymore.

https://my-website.scm.azurewebsites.net/letsencrypt/swagger returns 404.
Is this the correct/desired behaviour?

Apart from that problem thanks for the work that has been done. This extension is pretty good!

**Comment by sjkp** [comment]
Fixed in 0.9.1

**Comment by joelnotified** [comment]
Hi,
I just updated to 0.9.6 and I get an error on this page: https://sitename.scm.azurewebsites.net/letsencrypt/swagger/ui/index. When i click the error I get:: 

```
{
    schemaValidationMessages: [
    {
        level: "error",
        message: "Can't read from file https://sitename.scm.azurewebsites.net/letsencrypt/swagger/docs/2017-09-01"
    }
    ]
}
```

Should I change something?
